### PR TITLE
chore: adaptations for nightly-2024-06-29

### PR DESCRIPTION
**Pull request by jcommelin**

### CODE REVIEW

File: Mathlib/Combinatorics/SimpleGraph/Turan.lean [hunk 1]
```diff
@@ -124,10 +124,12 @@ lemma exists_isTuranMaximal :
 
 end Defs
 
-namespace IsTuranMaximal
+section Forward
```
Comment (jcommelin) [inline Mathlib/Combinatorics/SimpleGraph/Turan.lean:127]:
```suggestion
namespace IsTuranMaximal
```

File: Mathlib/Combinatorics/SimpleGraph/Turan.lean [hunk 2]
```diff
@@ -124,10 +124,12 @@ lemma exists_isTuranMaximal :
...
 
 variable {s t u : V} (h : G.IsTuranMaximal r)
 
+namespace IsTuranMaximal
+
```
Comment (jcommelin) [inline Mathlib/Combinatorics/SimpleGraph/Turan.lean:132]:
```suggestion
```

File: Mathlib/Combinatorics/SimpleGraph/Turan.lean [hunk 3]
```diff
@@ -296,4 +298,6 @@ theorem isTuranMaximal_iff_nonempty_iso_turanGraph (hr : 0 < r) :
     G.IsTuranMaximal r ↔ Nonempty (G ≃g turanGraph (Fintype.card V) r) :=
   ⟨fun h ↦ h.nonempty_iso_turanGraph, fun h ↦ isTuranMaximal_of_iso h.some hr⟩
 
+end Forward
+
```
Comment (jcommelin) [inline Mathlib/Combinatorics/SimpleGraph/Turan.lean:302]:
```suggestion
```

File: lakefile.lean [hunk 4]
```diff
@@ -18,12 +18,14 @@ package mathlib where
...
 
-require batteries from git "https://github.com/leanprover-community/batteries" @ "main"
-require Qq from git "https://github.com/leanprover-community/quote4" @ "master"
-require aesop from git "https://github.com/leanprover-community/aesop" @ "master"
-require proofwidgets from git "https://github.com/leanprover-community/ProofWidgets4" @ "v0.0.38"
-require Cli from git "https://github.com/leanprover/lean4-cli" @ "main"
-require importGraph from git "https://github.com/leanprover-community/import-graph.git" @ "main"
+meta if get_config? doc = some "on" then -- do not download and build doc-gen4 by default
+require «doc-gen4» from git "https://github.com/leanprover/doc-gen4" @ "main"
+
```
Comment (fpvandoorn) [inline lakefile.lean:23]:
```suggestion
```

Comment (semorrison) [reply lakefile.lean:23]: This was a bad merge. We just removed this on master.

**Comment by semorrison** [comment]
bors d+

**Comment by jcommelin** [comment]
Closing since

>  chore: adaptations for nightly-2024-06-30 #14306 

is already merged

**Pull request closed without merging**